Chapter 5 (Python loops)/task_3.py

```diff
@@ -0,0 +1,6 @@
+#Task 3: Loop through a Range with Step
+for number in range(10, 31, 3):
+    print(number)
+
+sum_numbers = sum(range(10, 31, 3))
+print(f"Sum: {sum_numbers}")
```

Comment:
```suggestion
sum_numbers = 0
for number in range(10, 31, 3):
    sum_numbers += number
    print(number)

print(f"Sum: {sum_numbers}")
```

This way we will avoid creating two identical instances of range